src/Page/Partners.elm

```diff
@@ -115,23 +115,29 @@ viewPartner partner =
 
 viewMap : List Data.PlaceCal.Partners.Partner -> Html msg
 viewMap partnerList =
+    let
+        allowOnlyPartnersWithLocation =
+            \partner ->
+                List.isEmpty partner.areasServed && (partner.maybeGeo /= Nothing)
+
+        partnerToGeo =
+            \partner ->
```

Comment:
```suggestion
        partnerToGeo partner =
```

Reply:
This is how the elm formatter likes it

Reply:
ah i see it now. we want it to be a lambda that takes one argument so List.filter/map can use it

Reply:
I think you misunderstand how this works in Elm. It's perfectly legal (and normal) to pass functions around, not lambdas. This lambda assignment looks odd and isn't something we do elsewhere in this codebase.

Reply:
The point of the lambda / anonymous function is to be able to define a function inline, which we're no longer doing by refactoring in this way.

Reply:
Perhaps I am committing a faux pa by using a lisp-ism but it is okay to define a lambda and assign it to a variable and then pass that variable in to a HLF like map. 

I changed it and it works so I guess you can define helper methods multiple ways.